ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/rtb/VungleRtbBannerAd.java

```diff
@@ -108,7 +109,12 @@ public void onInitializeSuccess() {
 
                 bannerAd = new BannerAd(context, placementForPlay, adConfig);
                 bannerAd.setAdListener(VungleRtbBannerAd.this);
-
+                String watermark = mediationBannerAdConfiguration.getWatermark();
+                if (!watermark.isEmpty()) {
```

Comment:
Better to use `!TextUtils.isEmpty(watermark)` to avoid NPE.

Reply:
Ok.